grails-app/conf/UrlMappings.groovy

```diff
@@ -68,6 +68,9 @@ class UrlMappings {
 		"/api/v1/users/me/keys"(resources: "keyApi", excludes: ["create", "edit", "update"]) { resourceClass = SecUser }
 		"/api/v1/users/me/products"(controller: "productApi", action: "index") { operation = Permission.Operation.SHARE }
 
+		"/api/v1/profile/changePassword"(controller: "profileApi", action: "changePassword", excludes: ["update"])
```

Comment:
These should be under `/api/v1/users/me/...` as that is the prefix which points to the current (logged in) user.

- `POST /api/v1/users/me/changePassword` (special functionality maybe warrants verb endpoint)
- `PUT /api/v1/users/me` for update (REST convention)